op-chain-ops/cmd/celo-migrate/main.go

```diff
@@ -1,42 +1,31 @@
 package main
 
 import (
-	"bytes"
 	"context"
 	"errors"
 	"fmt"
 	"math/big"
 	"os"
-	"path/filepath"
-	"time"
+	"os/exec"
+	"runtime/debug"
 
 	"github.com/ethereum-optimism/optimism/op-chain-ops/genesis"
 	"github.com/ethereum-optimism/optimism/op-service/jsonutil"
 	oplog "github.com/ethereum-optimism/optimism/op-service/log"
-	"github.com/ethereum-optimism/optimism/op-service/predeploys"
-	"github.com/holiman/uint256"
-	"github.com/mattn/go-isatty"
-
-	"github.com/urfave/cli/v2"
-
-	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/core"
-	"github.com/ethereum/go-ethereum/core/rawdb"
-	"github.com/ethereum/go-ethereum/core/state"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/ethclient"
-	"github.com/ethereum/go-ethereum/ethdb"
 	"github.com/ethereum/go-ethereum/log"
-	"github.com/ethereum/go-ethereum/params"
 	"github.com/ethereum/go-ethereum/rpc"
-	"github.com/ethereum/go-ethereum/trie"
-	"github.com/ethereum/go-ethereum/triedb"
+
+	"github.com/mattn/go-isatty"
+	"github.com/urfave/cli/v2"
+	"golang.org/x/exp/slog"
 )
 
 var (
 	deployConfigFlag = &cli.PathFlag{
 		Name:     "deploy-config",
-		Usage:    "Path to the JSON file that was used for the bedrock contracts deployment. A test example can be found here 'op-chain-ops/genesis/testdata/test-deploy-config-full.json' and documentation for the fields is at https://docs.optimism.io/builders/chain-operators/management/configuration",
+		Usage:    "Path to the JSON file that was used for the l1 contracts deployment",
```

Comment:
This change (and others below) change usage texts that Piers added.

Reply:
Ah, I assumed these were left over from the optimism bedrock migration script. 

Either way, I think the edits are accurate since we aren't migrating anything related to OP Bedrock and that could be a point of confusion.

@piersy let me know if there's anything about these edits I should change / revert

Reply:
Well, I would keep the extended usage string, since it provides useful extra context. Renaming bedrock -> l1, seems like a good move though.

Reply:
Okay awesome, added it back